x-pack/plugins/observability/public/components/shared/page_template/page_template.tsx

```diff
@@ -71,17 +72,30 @@ export function ObservabilityPageTemplate({
               exact: !!entry.matchFullPath,
               strict: !entry.ignoreTrailingSlash,
             }) != null;
-
+          const badgeLocalStorageId = `nav_item_badge_visible_${entry.app}${entry.path}`;
```

Comment:
We normally use `apm.`

https://github.com/elastic/kibana/blob/db9407c47ca829d35985db295f270a645f9813ac/x-pack/plugins/apm/public/components/app/Settings/schema/index.tsx#L34

Perhaps we should do the same here:

```suggestion
          const badgeLocalStorageId = `observability.nav_item_badge_visible_${entry.app}${entry.path}`;
```